README.md

```diff
@@ -1 +1,88 @@
-# angular-concepts-readme
+# Concepts
+
+## Overview
+
+There are some core concepts that every Angular developer will need to familiarise themselves with. This will teach you the core concepts that you will be using in every Angular application you make.
```

Comment:
Try to begin with a friendlier introduction to draw the reader in here. "In this lesson, we'll be looking at some of the core concepts of Angular and how they differ from Rails. Some will be familiar to you and some will be different - don't worry if it takes some time to sink in! "